databases/week1/week1-homework.sql

```diff
@@ -0,0 +1,48 @@
+-- 1- Find out how many tasks are in the task table.
+select count(*) as Total_number_of_task from task;
+-- 2- Find out how many tasks in the task table do not have a valid due date.
+select sum(case when due_date IS NULL then 1 else 0 end) as NUMBER_OF_NULL_VALUE from task;
+select * from task where due_date is null;
```

Comment:
Here you are missing the count, it should be `select count(*) from task where due_date is null;`